ui/components/core/src/lib/types/datasetTypes.ts

```diff
@@ -139,6 +140,19 @@ export interface ItemFeature {
   required?: boolean;
 }
 
+export interface FeaturesAvailableValues {
+  [key: string]: {
+    [innerKey: string]: Array<string>;
```

Comment:
your index here is not a variable, I think its more idiomatic to call always the same 

```

export interface FeaturesAvailableValues {
  [key: string]: {
    [key: string]: Array<string>;
  };
}
```